pkg/pillar/cmd/zedrouter/dnsmasq.go

```diff
@@ -489,12 +489,22 @@ func checkAndPublishDhcpLeases(ctx *zedrouterContext) {
 			ulStatus := &status.UnderlayNetworkList[i]
 			var assigned bool
 			var leasedIP net.IP
+			var snoopedIPs []net.IP
```

Comment:
Names could be more clear since the leased is leasedIPv4 and the snooped are snoopedPv6s

Reply:
Done. Changed names as suggested.